src/screens/Departures/state/stop-place-state.ts

```diff
@@ -23,7 +23,12 @@ import {flatMap} from 'lodash';
 import {EstimatedCall, Place} from '@atb/api/types/departures';
 import {getSecondsUntilMidnightOrMinimum} from './quay-state';
 
-const DEFAULT_NUMBER_OF_DEPARTURES_PER_QUAY_TO_SHOW = 5;
+export const DEFAULT_NUMBER_OF_DEPARTURES_PER_QUAY_TO_SHOW = 5;
+
+//We fetch double the number of departures to be shown,
+// so that we have more departures to show when one or more departures have already passed
```

Comment:
Excuse my OCD 😅

```suggestion
// We fetch double the number of departures to be shown, so that we have more 
// departures to show when one or more departures have already passed.
```

Reply:
hehehe..much cleaner 👍